_posts/2021-04-03-fix-warning-using-the-last-argument-as-keyword-parameters-is-deprecated.md

```diff
@@ -0,0 +1,104 @@
+---
+layout: post
+title:  "Fix: 'warning: Using the last argument as keyword parameters is deprecated'"
+date:   2021-04-03 08:00:00 +0100
+author: "Shadi Rezek"
+categories: techtips ruby
+og_image: ""
```

Comment:
please remove `og_image` if you don't use the image to avoid rendering empty `src` value in `<img src="" />`  on the blog homepage.